rekuho/src/lib/Portion.svelte

```diff
@@ -0,0 +1,177 @@
+<!--
+    Portion, may be simply the Editor
+    This window contains the visible and editable text.
```

Comment:
This comment seems a bit confusing to me, I only understand because we talked about this yesterday

Reply:
It confuses me too.